charts/ingress-nginx/templates/controller-role.yaml

```diff
@@ -91,7 +91,7 @@ rules:
       - list
       - watch
       - get
-{{- if .Values.podSecurityPolicy.enabled }}
+{{- if and .Values.podSecurityPolicy.enabled (not .Values.global.podSecurityStandards.enforced) }}
```

Comment:
As Giant Swarm is dropping PSP in recent releases and upstream is planning on doing so, that diff will probably disappear in the near future.